.gitignore

```diff
@@ -61,3 +61,5 @@ typings/
 .next
 .DS_Store
 dist/
+package-lock.json
```

Comment:
Remove the package-lock.json file. I have sent a guide on how to resolve conflicts on it in the channel. Check it out.

Reply:
And push it again.
cc @Bakley